Library/Homebrew/cmd/info.rb

```diff
@@ -401,6 +411,62 @@ def info_cask(cask)
 
         Cask::Info.info(cask, args:)
       end
+
+      sig { params(title: String, items: T::Array[T::Hash[Symbol, T.any(String, Integer)]]).void }
+      def print_sizes_table(title, items)
+        ohai title
```

Comment:
```suggestion
      def print_sizes_table(title, items)
        return if items.blank?

        ohai title
```
and could then skip the `formulae.present?` check below.